executioncontext/projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/model/ProjectView.kt

```diff
@@ -28,6 +21,10 @@ data class ProjectView constructor(
     val javaPath: ProjectViewJavaPathSection?,
     /** bazel flags added to all bazel command invocations  */
     val buildFlags: ProjectViewBuildFlagsSection?,
+    /** directories included and excluded from the project  */
+    val directories: ProjectViewDirectoriesSection?,
+    /** if set to true, relevant project targets will be automatically derived from the `directories` */
+    val deriveTargetsFlag: ProjectViewDeriveTargetsFromDirectoriesSection?,
```

Comment:
ohh i missed this place, here we'd like to keep the convention as well
```suggestion
    val deriveTargetsFromDirectoriesSection ProjectViewDeriveTargetsFromDirectoriesSection?,
```

Reply:
Changed var name